app/assets/stylesheets/administrate/application.scss

```diff
@@ -10,3 +10,5 @@
 @import "library/**/*";
 @import "base/**/*";
 @import "components/**/*";
+
+@import "administrate/customize";
```

Comment:
Not relevant, maybe?  The only CSS (other than the `customizations.scss` file) is the application manifest in the dummy app.  It does have that stupid sprockets `require_tree .` line, but I wasn't expecting the custom stuff in the main CSS file - I expect it to go in `administrate/application.css`, which I'm pretty sure it does.  After precompiling locally on my branch:

```
ag -A2 _customization /Users/keithlayne/forks/administrate/spec/example_app/public/assets/
/Users/keithlayne/forks/administrate/spec/example_app/public/assets/application-a07b8e973e3847f031ee869476ba76dedca6cc5f0979125f9f9b6330d60a210e.css
1:/* line 1, /Users/keithlayne/forks/administrate/spec/example_app/app/assets/stylesheets/administrate/_customizations.scss */
2-.custom-administrate-css-hidden {
3-  display: none; }

/Users/keithlayne/forks/administrate/spec/example_app/public/assets/administrate/application-951dc304ceab1b29c0253d06475d35bd25e97881066a6af900117a0241853739.css
1975:/* line 1, /Users/keithlayne/forks/administrate/spec/example_app/app/assets/stylesheets/administrate/_customizations.scss */
1976-.custom-administrate-css-hidden {
1977-  display: none; }
```

That disagrees with your results - could there be some other difference?